tests/foreman/ui/test_webhook.py

```diff
@@ -0,0 +1,81 @@
+"""Test class for Webhook UI
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: Webhooks
```

Comment:
Please include this component in testimony.yaml to fix failing GHA

Reply:
Just waiting on https://github.com/SatelliteQE/robottelo/pull/9741 and I'll rebase off that because I still need review on the airgun PR before this can be approved.

Reply:
@Griffin-Sullivan @mshriver done